docs/source/getting-started/authenticators-users-basics.md

```diff
@@ -25,10 +25,12 @@ If this configuration value is not set, then **all authenticated users will be a
 ```{note}
 As of JupyterHub 2.0, the full permissions of `admin_users`
 should not be required.
-Instead, you can assign [roles][] to users or groups
```

Comment:
What do you think about hyperlinking this to https://jupyterhub.readthedocs.io/en/stable/rbac/roles.html#define-role-target instead of having the separate link below?

Reply:
tried doing that but the hyperlinks were not working inside the note box

Reply:
tried doing that but the hyperlinks were not working inside the note box

Reply:
> tried doing that but the hyperlinks were not working inside the note box

Hmm, that is strange. According to the [`myst parser docs`](https://myst-parser.readthedocs.io/en/latest/syntax/roles-and-directives.html#how-directives-parse-content), markdown links should render inside a `note` directive. There is a `markdown link syntax` example in the docs I linked. It's inside an `{admonition}` directive, but it should work inside a `note` one too.

@Uzor13, do you think you can try again, in case there was a tiny syntax error the first time? thank you 🚀

Reply:
@GeorgianaElena thanks, that worked